api/aperture/cloud/v1/cmd.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
```

Comment:
please rename also the file (either cloud.proto or policy.proto or sth else is fine. Cmd is now unrelated)

Reply:
Done